README.adoc

```diff
@@ -112,7 +108,7 @@ my-typescript-project
 └── vite.config.ts
 ----
 <1> After Vite compiles and builds your project, the `dist` folder will contain your final system or module in its entirety _(i.e. what you'll actually distribute to users)_.
-<2> The `static` folder contains project files that don't require any compiling or processing before they are used, such as images, fonts, localization files, Handlebars templates, etc. These files will be copied to the `dist` folder as-is during the build process. _(I've included a few examples, which may or may not be relevant to your project.)_
+<2> The `static` folder contains project files that don't require any compiling or processing before they are used, such as images, fonts, localization files, Handlebars templates, etc. These files will be copied to the `dist` folder as-is during the build process. _(A few examples have been included, which may or may not be relevant to your project.)_
```

Comment:
I got rid of any authorial pronouns because I believe that this README will be a living document with many authors over time. This means no disrespect towards you.

In truth I was tempted to rewrite everything avoiding pronouns like 'you', 'we', etc. However I decided that the tone was decidedly informal enough that it wasn't necessary.

Reply:
Yeah I vacillated on how formal a tone to use quite a bit, so it'll likely take some time before we're on the exact same page in that regard --- and no disrespect taken, you're absolutely right about this being a living document with no single author!

I do think a slightly informal tone strikes the best balance -- pretty much what you described, removing the author references but retaining the second-person pronouns.  (I'll do a once-over for "we" and remove that too, just to be consistent).